src/components/search/SearchResultsPaging.jsx

```diff
@@ -29,6 +27,7 @@ const SearchResultsPaging = (props) => {
    *    › - next
    *   » - last
    */
+  /* eslint no-unused-vars: 'off' */
   const handleClick = (event) => {
```

Comment:
Can't you just omit the event?

Reply:
Done, implementation should be part of the work for ticket #1518